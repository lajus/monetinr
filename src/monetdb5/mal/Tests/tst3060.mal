# The direction for future expansion of the properties
function tst(a:any_1{type="int,flt"}):any_1;
	io.print(a);
	return tst:=a;
end tst;

user.tst(1);
user.tst("wrong");
