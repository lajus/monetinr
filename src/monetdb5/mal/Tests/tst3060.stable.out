stdout of test 'tst3060` in directory 'monetdb5/mal` itself:


# 14:57:24 >  
# 14:57:24 >  Mtimeout -timeout 60 mserver5 "--config=/ufs/mk/monet5/Linux/etc/MonetDB5.conf" --debug=10 --set "monet_mod_path=/ufs/mk/monet5/Linux/lib/MonetDB5:/ufs/mk/opensource/MonetDB/Linux/lib/MonetDB" --set "gdk_dbfarm=/ufs/mk/monet5/Linux/var/MonetDB5/dbfarm" --set "sql_logdir=/ufs/mk/monet5/Linux/var/MonetDB5/dblogs" --set mapi_port=31801 --set monet_prompt= --trace --dbname=mTests_src_mal  tst3060.mal < /dev/null
# 14:57:24 >  

# MonetDB Server v4.99.19
# Copyright (c) 1993-2006 CWI, all rights reserved
# Compiled for x86_64-redhat-linux-gnu/64bit with 64bit OIDs dynamically linked
# config:/ufs/mk/monet5/Linux/etc/MonetDB5.conf
# dbfarm:/ufs/mk/monet5/Linux/var/MonetDB5/dbfarm
# dbname:mTests_src_mal
# Visit http://monetdb.cwi.nl/ for further information
## The direction for future expansion of the properties
#function tst(a:any_1{type="int,flt"}):any_1;
#	io.print(a);
#	return tst:=a;
#end tst;
function user.tst(a:any_1):any_1;
    io.print(a);
    return tst := a;
end tst;
#
#user.tst(1);
#user.tst("wrong");
#
function user.main():void;
# The direction for future expansion of the properties 
    user.tst(1);
    user.tst("wrong");
end main;
[ 1 ]
[ "wrong" ]

# 14:57:25 >  
# 14:57:25 >  Done.
# 14:57:25 >  

